testgrid/config/config.yaml

```diff
@@ -174,6 +174,8 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-gci-ci-slow-release-1.5
 - name: ci-kubernetes-e2e-gci-gce-examples
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-examples
+- name: ci-kubernetes-e2e-gce-gci-qa-docker13
```

Comment:
Consider adding these to the `google-docker` tab, which was intended for docker validation tests